guillotina/api/content.py

```diff
@@ -212,6 +212,13 @@ async def __call__(self):
         for behavior in behaviors or ():
             obj.add_behavior(behavior)
 
+        # Local Roles assign owner as the creator user
```

Comment:
Why move this up?

Reply:
If owner is not set before deserialization field permissions are not available as guillotina.Owner on the specific object